test/test_other.py

```diff
@@ -8885,7 +8885,11 @@ def do_test(cmd):
 
   @requires_v8
   def test_single_file_shell(self):
-    self.do_runf('hello_world.c', emcc_args=['-sSINGLE_FILE'])
```

Comment:
How did this pass before? `requires_v8` should have run the test in v8, and without building with `ENVIRONMENT=shell` that should have failed?

Reply:
requires_v8 injects `-sENVRIONMENT=shell`.. but in this case I thought it better to be explicit.. maybe not ?

Reply:
In that case I'd remove it. It confused me, and also it would be confusing in the logging to have the flag appear twice.

Reply:
Done